e2e/types.ts

```diff
@@ -1,13 +1,22 @@
 declare global {
     interface Window {
         callsClient: any,
-        desktop: any,
         isHandRaised: boolean;
         e2eDesktopNotificationsRejected?: DesktopNotificationArgs[],
         e2eDesktopNotificationsSent?: string[],
         e2eNotificationsSoundedAt?: number[],
         e2eNotificationsSoundStoppedAt?: number[],
         e2eRingLength?: number,
+
+        // Desktop Mocking
+        desktop: any,
```

Comment:
You probably don't need this anymore, it's unfortunately still there because of the `version` field, which has been replaced by `getAppInfo` under the `desktopAPI`